lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -75,6 +75,7 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
     'crop_width' => $values['crop_width'],
     'crop_height' => $values['crop_height'],
     'crop_rotate' => $values['crop_rotate'],
+    'source' => isset($values['source']) ? $values['source'] : NULL,
```

Comment:
@sbsmith86 As of now I left this part but pass source from elsewhere in phoenix. I figured that once the data is in this function we might not know for sure that it came from phoenix web because I'm guessing we will be using this function when sending over data that was sent to the phoenix api. What are your thoughts?